File: Retrofit/app/src/main/java/com/okidoki/android/retrofit/dog_app/java/data/DogImages.java

package com.okidoki.android.retrofit.dog_app.java.data;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class DogImages {

    @SerializedName("status")
    @Expose
    public String status;
    @SerializedName("message")
    @Expose
    public List<String> message;

    @Override
    public String toString() {
        return "DogImages{" +
                "status='" + status + '\'' +
                ", message=" + message +
                '}';
    }
}

File: Retrofit/app/src/main/java/com/okidoki/android/retrofit/dog_app/kotlin/data/DogImages.kt
package com.okidoki.android.retrofit.dog_app.kotlin.data

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class DogImages {

    @SerializedName("status")
    @Expose
    var status: String? = null
    @SerializedName("message")
    @Expose
    var message: List<String>? = null

    override fun toString(): String {
        return "DogImages{" +
                "status='" + status + '\''.toString() +
                ", message=" + message +
                '}'.toString()
    }
}
